digits/templates/partials/home/pretrained_model_pane.html

```diff
@@ -0,0 +1,126 @@
+<!-- Pretrained Model -->
+<div class="col-md-12" ng-show="tab.isSet(3)">
+    <div class="well">
+        <div ng-controller="pretrained_models_controller">
+            <div class="row">
+                <div class="col-md-12">
+                    <!-- Output Field Options -->
+                    <div class="button-group pull-right">
+                        <button type="button" class="btn btn-default btn-sm dropdown-toggle" data-toggle="dropdown">
+                            <span class="glyphicon glyphicon-cog"></span>
+                            <span class="caret"></span>
+                        </button>
+                        <ul class="dropdown-menu" ng-click="$event.stopPropagation()">
+                            <li data-ng-repeat='field in storage.pretrained_model_fields'>
+                                <label style="font-weight: normal !important;">
+                                    <input type="checkbox" ng-model='field.show'>
+                                    {[ field.name ]}
+                                </label>
+                            </li>
+                            <li data-ng-repeat='field in storage.model_output_fields'>
```

Comment:
The files downloaded from the Download Model button include the mean.binaryproto, the prototxt files, caffemodel, and the newly added info.json. I don't see anything that would downloaded containing data in the model_output_fields. Perhaps that was not intended to there?